napari/utils/settings/_defaults.py

```diff
@@ -202,7 +202,7 @@ class AppearanceSettings(BaseNapariSettings):
     )
 
     highlight_thickness: HighlightThickness = Field(
-        1,
+        2,
```

Comment:
Can this be 
```suggestion
        HighlightThickness.highlight_thickness,
```
to avoid the duplication of the `2`, or does that not work syntactically inside the `Field`?

Reply:
Great suggestion!  It works for me.  :)